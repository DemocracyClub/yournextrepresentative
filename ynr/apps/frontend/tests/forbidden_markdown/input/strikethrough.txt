Strikethrough uses two tildes. ~~Scratch this.~~
