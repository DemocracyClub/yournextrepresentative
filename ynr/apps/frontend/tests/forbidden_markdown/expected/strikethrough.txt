<p>Strikethrough uses two tildes. ~~Scratch this.~~</p>
